cv32e40s/tests/cfg/pmp_g3r3.yaml

```diff
@@ -4,8 +4,170 @@ compile_flags:
     +define+PMP_G3R3
     +define+ZBA_ZBB_ZBC_ZBS
 ovpsim: >
-    --override root/cpu/PMP_grain=3
-    --override root/cpu/PMP_registers=3
+    --override cpu/PMP_grain=3
+    --override cpu/PMP_registers=64
+    --override cpu/PMP_initialparams=T
+    --override cpu/PMP_maskparams=T
+    --override cpu/pmpaddr0=0
+    --override cpu/pmpaddr1=0
+    --override cpu/pmpaddr2=0
```

Comment:
If there should be 3 regions in this config, should the first 3 overrides here not exist?

Reply:
Or is that handled by the masks below?

Reply:
pmpaddrX sets default value, mask_pmpaddrX sets writeable bitmask. The RTL expects all 64 registers to always be present, even when PMP number of regions is set to 0 - this fix works around that in the ISS, which completely disables the PMP regions if number of regions = 0

Reply:
Good